WordPress/Classes/Networking/ServiceRemoteWordPressComREST.m

```diff
@@ -59,4 +59,17 @@ - (NSString *)pathForEndpoint:(NSString *)resourceUrl
     return [NSString stringWithFormat:@"%@/%@", apiVersionString, resourceUrl];
 }
 
++ (WordPressComRestApi *)anonymousWordPressComRestApi {
+    static WordPressComRestApi *_anonymousWordPressComRestApi = nil;
```

Comment:
Why is this static? An anonymous API is mostly used when there's no account for sign in or sign up. As soon as an account is added, it's not likely to be used again. I don't see the value on keeping it in memory.